liquibase-core/src/main/java/liquibase/util/StringClauses.java

```diff
@@ -501,7 +501,7 @@ public boolean isEmpty() {
             return true;
         }
         for (Object clause : clauses.values()) {
-            if ((clause != null) && !"".equals(clause.toString().trim())) {
+            if (!"".equals(clause.toString().trim())) {
```

Comment:
This style of preventing null pointers is less readable than using `isEmpty`. Can you please see if this code can be cleaned up to use `isEmpty`?

Reply:
Hey @XDelphiGrl, yes we could make this change, but I would personally recommend doing it in a separate PR as it would be a separate readability improvement not related to changes introduced here. I just look for a comparison like this one and there are a couple of places where we need to make this update.

Reply:
Makse sense to me. This code serves the same purpose, it is just a bit of an outdated approach.

Reply:
@XDelphiGrl (cc: @filipelautert ) I totally agree it would make it a bit more readable what you are suggesting. I can quickly make a separate PR and make this update in a couple of files I saw yesterday we might need to update.

Reply:
@XDelphiGrl created this [issue](https://github.com/liquibase/liquibase/issues/3718) to address empty string check.